CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/CallConfiguration.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Azure.Communication.Identity;
+using Azure.Communication;
+
+namespace CallAutomation_AppointmentReminder
+{
+    /// <summary>
+    /// Configuration assoociated with the call.
+    /// </summary>
+    public class CallConfiguration
+    {
+        public CallConfiguration()
+        {
+
+        }
+
+        /// <summary>
+        /// The connectionstring of Azure Communication Service resource.
+        /// </summary>
+        public string ConnectionString { get; set; }
+
+        /// <summary>
+        /// The source identity.
+        /// </summary>
+        public string SourceIdentity =>  CreateUser(this.ConnectionString).Result;
+
+        /// <summary>
+        /// The phone number to add to the call
+        /// </summary>
+        public string TargetPhoneNumber { get; set; }
+
+        /// <summary>
+        /// The phone number associated with the source. 
+        /// </summary>
+        public string SourcePhoneNumber { get; set; }
+
+        /// <summary>
+        /// The base url of the applicaiton.
+        /// </summary>
+        public string AppBaseUri { get; set; }
+
+        /// <summary>
+        /// The base url of the applicaiton.
+        /// </summary>
+        public string EventCallBackRoute { get; set; }
+
+        /// <summary>
+        /// The callback url of the application where notification would be received.
+        /// </summary>
+        public string CallbackEventUri => $"{AppBaseUri}" + EventCallBackRoute;
+
+        /// <summary>
+        /// Appointment reminder menu audio file route
+        /// </summary>
+        public string AppointmentReminderMenuAudio { get; set; }
+
+        /// <summary>
+        /// Appointment confirmed audio file route
+        /// </summary>
+        public string AppointmentConfirmedAudio { get; set; }
+
+        /// <summary>
+        /// Appointment cancelled audio file route
+        /// </summary>
+        public string AppointmentCancelledAudio { get; set; }
+
+        /// <summary>
+        /// Invalid input audio file route
+        /// </summary>
+        public string InvalidInputAudio { get; set; }
+
+        /// <summary>
+        /// Time out audio file route
+        /// </summary>
+        public string TimedoutAudio { get; set; }
+
+
+        public async Task<string> CreateUser(string connectionString)
```

Comment:
Could we separate this into it's own service maybe? Best practices would be to create a class, interface, register in DI, then inject where you need it. Another approach would be to use an extension method off the `WebApplication` class. Give it a name like `app.ProvisionAzureCommunicationServicesIdentity()` and inside the method use your identity client with a `.Wait()` method on the async operation. I can show you how to do this if you want help.